napalm/base.py

```diff
@@ -364,3 +364,44 @@ def get_interfaces_counters(self):
             }
         """
         raise NotImplementedError
+
+        def show_route(self, destination = ''):
```

Comment:
How about leaving that name, and return an exception when an IPv6 route is asked ?
I think ultimately we should be able to use the same function for v4 and v6.